CHANGELOG.md

```diff
@@ -1152,3 +1152,4 @@
 [@fwolfst]: https://github.com/fwolfst
 [@maxprokopiev]: https://github.com/maxprokopiev
 [@ytjmt]: https://github.com/ytjmt
+[@cdudas17]: https://github.com/cdudas17
```

Comment:
This file is autogenerated, please remove. When a release is made it is properly updated.